docs/content/guides/kamaji-azure-deployment.md

```diff
@@ -211,6 +213,8 @@ kind: Service
 metadata:
   name: ${TENANT_NAME}-public
   namespace: ${TENANT_NAMESPACE}
+  labels:
+    tenant.clastix.io: ${TENANT_NAME}
```

Comment:
This is a manifest for a `Service`, we don’t need to label it according to the issue

Reply:
Done

Reply:
once again, we don’t need to label services